hunting-buddy.lic

```diff
@@ -226,6 +227,11 @@ class HuntingBuddy
 
   def hunt(args, duration, stop_on_skills, stop_on_low_skills, stop_on_boxes)
     $COMBAT_TRAINER = nil
+    hunting_room = Room.current.id
+    #echo("hunting-room number is #{hunting_room}")
```

Comment:
How about uncommenting all of the echos and changing them to  `echo("hunting-room number is #{hunting_room}") if $debug_mode_ct` for example?

Reply:
its defined here https://github.com/rpherbig/dr-scripts/blob/26d6e394d2b5f1da179fb6122c1f0a107f4f2808/combat-trainer.lic#L2525 and used throughout the whole script for the same thing https://github.com/rpherbig/dr-scripts/blob/26d6e394d2b5f1da179fb6122c1f0a107f4f2808/combat-trainer.lic#L13

Reply:
That is in combat-trainer, not hunting-buddy. I assume then it is passed?

Reply:
oh crap, no, I'm sorry. I thought this was all being handled in combat-trainer for some reason! the echos should just be deleted then, I'm sorry about that Dart. teach me to review anything on a phone :)

Reply:
:)